Project.toml

```diff
@@ -5,12 +5,14 @@ version = "0.7.1"
 [deps]
 AbstractFFTs = "621f4979-c628-5d54-868e-fcf4e3e8185c"
 DSP = "717857b8-e6f2-59f4-9121-6e50c889abd2"
+Infinities = "e1ba4f0e-776d-440f-acd9-e1d2e9742647"
```

Comment:
Infinities shouldn't be a dependency. Maybe you want it as a test dependency?

Reply:
Infinity is used for Hankel construction from AbstractMatrix. If the matrix is infinite then the last row/column is unavailable.

Reply:
To work around, we could modify InfiniteArrays.jl so that they can somehow "return" the last row/column. Then we let `vcat(a::InfiniteVector, b::AbstractVector)=a` and the Hankel can safely drop the dependency.